tox.ini

```diff
@@ -33,6 +33,19 @@ commands =
   python -m sphinx -v -b doctest -d doctrees docs html
   find html -type f -name "*.ipynb" -not -path "html/_sources/*" -delete
 
+[testenv:docs-clean]
+description = Clean all the existing built objects and build them from scratch
+deps = -r requirements/docs.txt
+allowlist_externals = find
+setenv =
+  {[testenv]setenv}
+  # Search directory for confuse for custom config enabling plopp plotting
+  SCIPPDIR={toxinidir}{/}docs{/}buildconfig
+commands =
+  python -m sphinx -v -b html -d doctrees docs html -E
```

Comment:
Are you sure `-E` is sufficient for all possible changes? Does Sphinx regenerate everything? I remember having trouble in the past.

Reply:
I tried updating only `css` file and it worked so I was just curious why it wouldn't be sufficient.
I will add deleting lines again just in case.